### PR TITLE
Fix dartanalyzer

### PR DESCRIPTION
**Pull request by jonasfj**
Fixed dartanalyzer issues, I left the two test issues for @cskau-g  to finish :)

### CODE REVIEW

File: test/src/flowcontrol/stream_queues_test.dart [hunk 1]
```diff
@@ -132,7 +131,7 @@ main() {
       expect(queue.bufferIndicator.wouldBuffer, isTrue);
       // We assert that we got the data, but it wasn't processed.
       verify(windowMock.gotData(bytes.length)).called(1);
-      verifyNever(windowMock.dataProcessed(typed(any)));
+      verifyNever(windowMock.dataProcessed(any));
```
Comment (mkustermann) [inline test/src/flowcontrol/stream_queues_test.dart]: Is that due to using a new version of `package:mockito`?

**Comment by jonasfj** [comment]
Formatting was fixed (I think), I took a few pushes to hit the right branch :)

Currently, travis is only annoyed by the test failures, which were already present on master. I think @cskau-g  is working on those.

**Comment by mkustermann** [comment]
Ship it!

**Comment by cskau-g** [comment]
🛳

**Pull request merged**